orchestrator/oracle/oracle.go

```diff
@@ -59,16 +62,23 @@ func New(ctx context.Context, logger zerolog.Logger, providersName []string) (*O
 		}
 	}
 
-	oracle := &Oracle{
-		logger:                logger.With().Str("module", "oracle").Logger(),
-		closer:                ummedpfsync.NewCloser(),
-		providers:             providers,
-		subscribedBaseSymbols: map[string]struct{}{},
+	o := &Oracle{
+		logger:                  logger.With().Str("module", "oracle").Logger(),
+		closer:                  ummedpfsync.NewCloser(),
+		providers:               providers,
+		subscribedBaseSymbols:   map[string]struct{}{},
+		providerSubscribedPairs: map[string][]umeedpftypes.CurrencyPair{},
 	}
-	oracle.loadAvailablePairs()
-	go oracle.start(ctx)
+	o.loadAvailablePairs()
+	if err := o.subscribeProviders([]umeedpftypes.CurrencyPair{
+		{Base: "USDT", Quote: "USD"},
+		{Base: "DAI", Quote: "USD"},
```

Comment:
Are there consts we can use instead of inline hardcoding?

Reply:
good, I will create those